docs/downloads/single-file-example.html

```diff
@@ -11,16 +11,9 @@
     <div id="root"></div>
     <script type="text/babel">
 
-      class Hello extends React.Component {
-        render() {
-          return <h1>Hello, world!</h1>;
-        }
-      }
-
-      ReactDOM.render(
-        <Hello />,
-        document.getElementById('root')
-      );
+      const element = <h1>Hello, world!</h1>;
+      const container = document.getElementById('root');
```

Comment:
i would call this "root" rather than container to avoid confusing it with container components. or just put it inline in the render call.